inputs/values/dockers.json

```diff
@@ -2,7 +2,7 @@
   "name": "dockers",
   "cnmops_docker": "us.gcr.io/broad-dsde-methods/gatk-sv/cnmops:2023-07-28-v0.28.1-beta-e70dfbd7",
   "condense_counts_docker": "us.gcr.io/broad-dsde-methods/tsharpe/gatk:4.2.6.1-57-g9e03432",
-  "gatk_docker": "us.gcr.io/broad-dsde-methods/markw/gatk:2023-07-13-4.4.0.0-43-gd79823f9c-NIGHTLY-SNAPSHOT",
+  "gatk_docker": "us.gcr.io/broad-dsde-methods/markw/gatk:mw-sv-concordance-opt-1097258",
```

Comment:
Thank you for updating this image! To keep GCR and ACR in sync, I pushed the updated image to ACR; so could you please update `inputs/values/dockers_azure.json` with the following image? 

```
vahid.azurecr.io/markw/gatk:mw-sv-concordance-opt-1097258
```